CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [2.6.6]
+
+- add nrepl/nrepl 0.6.0 to move to mainline development for nrepl
```

Comment:
Updates tk to 2.0 to take advantage of the newest nrepl?

Reply:
Done!